PsychSourceGL/Cohorts/FTGLTextRenderer/libptbdrawtext_ftgl.cpp

```diff
@@ -450,11 +449,7 @@ void PsychSetTextBGColor(int context, double* color)
 // renderbackend needs a different geometric setup:
 void PsychSetTextViewPort(int context, double xs, double ys, double w, double h)
```

Comment:
I understand that, but without setting it, you just created another version of a SetviewPort that does nothing if one wants to actually use it, so it would be pointless.